profiles/common/modules/features/splash_screen/includes/splash_screen.admin.inc

```diff
@@ -63,6 +79,11 @@ function _splash_screen_admin_submit($form, &$form_state) {
   // Updates the $form_state before passing it to the Drupal system.
   $form_state['values']['splash_screen_blacklist'] = $blacklist_arr;
 
+  if (empty($form_state['values']['splash_screen_title_value']) && $form_state['values']['splash_screen_set_title'] == 1) {
```

Comment:
Verification should be done on a "validate" function ``_splash_screen_settings_form``